code/modules/reagents/reagent_containers.dm

```diff
@@ -404,7 +404,8 @@
 
 	var/trans = round(reagents.trans_to(target, amount_per_transfer_from_this, transferred_by = user), CHEMICAL_VOLUME_ROUNDING)
 	playsound(target.loc, SFX_LIQUID_POUR, 50, TRUE)
-	to_chat(user, span_notice("You transfer [trans] unit\s of the solution to [target]."))
+	if(trans)
```

Comment:
🏳️‍⚧️ ?

Reply:
I am 100% male and love Jesus. No transgender here